lib/Containers/MainContainer.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+
+// bpanel imports
+import { Header, TabMenu } from '@bpanel/bpanel-ui';
+
+// local imports
+import CreateWallet from './CreateWallet';
+import SelectWallet from './SelectWallet';
+import MultisigWallet from './MultisigWallet';
+import ActionContainer from './ActionWrapper';
+
+export default class MainContainer extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    // build tabs to render
+    const tabs = [
+      {
+        header: 'Select Wallet',
+        body: (
+          <ActionContainer>
+            <SelectWallet />
+          </ActionContainer>
+        ),
+      },
+      {
+        header: 'Create Wallet',
+        body: (
+          <ActionContainer>
+            <CreateWallet />
+          </ActionContainer>
+        ),
+      },
+      {
+        header: 'Multisig Wallet',
+        body: <MultisigWallet />,
```

Comment:
It looks like this component ends up just getting wrapped in `ActionContainer` in its own render method too, why not wrap it here like the others?

Reply:
The children components of the `MultisigWallet` need the methods injected by the `ActionContainer` whereas `CreateWallet`/`SelectWallet` consume the methods directly

Reply:
Ah got it. And those children need to be wrapped in a pointless `div` 😏. Actually I think a new version of React (maybe not released yet) no longer needs those to be wrapped which would mean you could pull the children up a level and wrap this the same as the others. But that's neither here nor there 👍